pkg/apiserver/logsearch/models.go

```diff
@@ -54,7 +54,30 @@ var ServerTypeMap = map[ServerType]string{
 	ServerTypePD:   "pd",
 }
 
-type SearchLogRequest diagnosticspb.SearchLogRequest
+type LogLevel int32
+
+type SearchLogRequest struct {
+	StartTime int64      `protobuf:"varint,1,opt,name=start_time,json=startTime,proto3" json:"start_time,omitempty"`
+	EndTime   int64      `protobuf:"varint,2,opt,name=end_time,json=endTime,proto3" json:"end_time,omitempty"`
+	Levels    []LogLevel `protobuf:"varint,3,rep,packed,name=levels,enum=diagnosticspb.LogLevel" json:"levels,omitempty"`
```

Comment:
Prefer to change it to `MaxLevel LogLevel`, since the front-end only accepts max level.